test/db-common.sh

```diff
@@ -8,6 +8,5 @@ function die() {
 
 SERVICES="sa
 policy"
-DBENVS="development
-test
+DBENVS="test
```

Comment:
Why for?

I'm guessing so it doesn't delete the db when you're devving? It'd be nice if we could at least set up the db correctly for folks (as the docs mention you can), but maybe we can't?